backend/Models/CreateSubtireddModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace backend.Models
+{
+    public class CreateSubtireddModel
+    {
+        [Required(ErrorMessage = "Subtiredd name is required")]
```

Comment:
Tak właściwie to potrzebujemy tych błędów? Każdy input model w nie będzie musiał być wyposażony, a w sumie jak tych wartości brakuje to i tak jest to raczej błąd aplikacji aniżeli użytkownika.

Reply:
Robiliśmy je już przy rejestracji w ramach podwójnej walidacji, więc uznałem, że dla spójności tutaj też je dodam